src/openeo_gfmap/fetching/generic.py

```diff
@@ -116,6 +116,10 @@ def generic_default_processor(cube: openeo.DataCube, **params):
             "fetch": partial(_get_generic_fetcher, collection_name="AGERA5"),
             "preprocessor": partial(_get_generic_processor, collection_name="AGERA5"),
         },
+        Backend.CDSE_STAGING: {
+            "fetch": partial(_get_generic_fetcher, collection_name="AGERA5"),
+            "preprocessor": partial(_get_generic_processor, collection_name="AGERA5"),
+        },
```

Comment:
Is this still needed? 

The last thing I heard was that AGERA5 would not be downloaded?